<launch>
  <node pkg="actionlib" type="axclient.py" args="/Behaviors/learningFromDemonstration" name="TrajectoryGeneratorInterfaceGui" output="screen"/>
</launch>
